js/src/ui/Form/TypedInput/typedInput.js

```diff
@@ -22,7 +22,10 @@ import IconButton from 'material-ui/IconButton';
 import AddIcon from 'material-ui/svg-icons/content/add';
 import RemoveIcon from 'material-ui/svg-icons/content/remove';
 
-import { Input, InputAddressSelect, Select } from '../../../ui';
+import Input from '../../../ui/Form/Input';
```

Comment:
Still one in `src/api/contract.js` importing `Api` to check that the given `api` in constructor is an instance of `Api` class, and `src/api/api.js` importing `Contract` used in the `newContract` method...